frontend/components/Settings/ProfileForm.tsx

```diff
@@ -80,6 +80,14 @@ const ProfileForm = ({
     }
   }
 
+  const affiliations = [
+    { value: 0, label: 'Undergraduate Student' },
+    { value: 1, label: 'Masters Student' },
+    { value: 2, label: 'Professional Student' },
+    { value: 3, label: 'Doctoral Student' },
+    { value: 4, label: 'Faculty or Staff Member' },
```

Comment:
@ArmaanT Rerequesting review so I can get your opinion on these labels. Are there better labels that we can use? What is the difference between a masters and a professional student? They generally seem very intertwined when Penn refers to them, would everyone be able to tell which one they should select?

Reply:
On platform we're using the labels that Penn used to label degrees ([link](https://catalog.upenn.edu/programs/)). I'm sure Penn isn't very consistent in their naming, but I think this should be good enough for us. There are definitely conflicts (for example an Accounting MBA is both a Master's and a Professional degree) so we just choose the first label that applies. Bachelor's, then Master's, then PhD, then Professional